packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -282,9 +284,13 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
   private _onInputChange(event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>): void {
     const element: HTMLInputElement = event.target as HTMLInputElement;
     const value: string = element.value;
-    if (value === this.state.value) {
+
+    // HACK: skip validation given both onInput and onChange are attached
```

Comment:
Can you adjust the language here, unless you expect this to be resolved in the future? We should avoid "hacks" in the code. In this case, you are avoiding unnecessary/redundant work to be done when the value has not changed.